vllm/model_executor/custom_op.py

```diff
@@ -69,3 +75,40 @@ def dispatch_forward(self):
             return self.forward_xpu
         else:
             return self.forward_cuda
+
+    @classmethod
+    def enabled(cls) -> bool:
+        enabled = f"+{cls.name}" in envs.VLLM_CUSTOM_OPS
```

Comment:
would it be simpler to just use the python class name and not have to register anything?

Reply:
The last commit adds a registry of custom ops which requires the registration anyway. So I'm inclined to keep it this way - but we could change it to class names to avoid different capitalization?

Reply:
What happens if someone forgets to register their CustomOp?  Will it use the global enable/disable settings?  

Also, is there a way to avoid the manual name setting in `FlippedSiluAndMul`?  Can you not use register on that class too?

Reply:
I assume it will fail as it won't find `cls.name` but I can improve the error message here.

Reply:
FlippedSiluAndMul is an oversight - it shouldn't be registered and should just use the SiluAndMul settings - but I'll verify that's how it works.